api/src/main/java/io/lumeer/engine/api/data/DataStorage.java

```diff
@@ -111,13 +111,13 @@ default void connect(final StorageConnection connection, final String database,
    /**
     * Creates and inserts a new document to specified collection.
     *
-    * @param collectionName
-    *       the name of the collection where the document will be created
+    * @param collectionCode
+    *       the code of the collection where the document will be created
     * @param document
     *       the DataDocument object representing a document to be created
     * @return the id of the newly created document
     */
-   String createDocument(final String collectionName, final DataDocument document);
+   String createDocument(final String collectionCode, final DataDocument document);
```

Comment:
Why this change? This is not about collections in our terminology but MongoDB collections which have names.